roles/openshift_sanitize_inventory/tasks/deprecations.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Check for usage of deprecated variables
+  debug: msg="{{ __deprecation_header }} {{ item }} is a deprecated variable and will be no longer be used in the next minor release. Please update your inventory accordingly."
+  when:
+  - lookup('env', item) | bool
+  with_items: "{{ __warn_deprecated_vars }}"
```

Comment:
So, we still see this called once per host and I'm not sure if we should change that process. If I try to use the pattern to singletonize that you had used for the excluder, I still see it run against each host...